src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
-body {
+$squere: 300px;
+
+.body {
   margin: 0;
 }
+
+.snake {
+  display: grid;
+  justify-content: center;
+  grid-template-areas: "block1" "block2" "block3" "block4" "block5" "block6";
```

Comment:
let's use grid-template-columns in this case with repeat function, so grid can handle it by itlself